packages/frontend/src/components/svg/SwapIconTwoArrows.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const SwapIconTwoArrows = ({ color, width, height, margin }) => {
+    const stroke = color || "#000";
+    const currentWidth = width || "20";
+    const currentHeight = height || "20";
+    const currentMargin = margin || "0px";
+    return (
+        <svg
+            style={{ marginTop: margin }}
+            height={currentHeight}
+            viewBox="0 0 50 50"
+            width={currentWidth}
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <g
+                fill={color}
+                fill-rule="evenodd"
+                stroke={color}
+                stroke-width="3.5"
+                stroke-linecap="round"
+                stroke-linejoin="round"
```

Comment:
Quick camel case update

```suggestion
                fillRule="evenodd"
                stroke={color}
                strokeWidth="3.5"
                strokeLinecap="round"
                strokeLinejoin="round"
```